### ISSUE TITLE
Missing white space when multiple cited ranges (at least when citing GEDSH)

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srp-places-app#80]
@paregorios Sorry to add this, I think it is new as I have never seen it before:

See this example (38-139Map I) on place/78:

Sebastian P. Brock et al. (ed.), The Gorgias Encyclopedic Dictionary of the Syriac Heritage (Piscataway, NJ: Gorgias Press, 2011), pp. 138-139Map I B1, II B1, III.

**Comment by davidamichelson** [comment]
@paregorios More feedback on this, it seems more complicated. It looks like place/78 is not showing the article from bibl/1.

**Comment by davidamichelson** [comment]
@tacarlson @paregorios Actually looks like it might be an issue in the TEI. Thomas?

**Comment by davidamichelson** [comment]
http://srophe.github.io/srp-places-app/places/8.html on the other hand is working perfectly.

**Comment by davidamichelson** [comment]
Hmm, well I think we might need to talk in person (and maybe we wait until the next milestone since what is there now mostly works.)

I wonder if each GEDSH article actually needs to be a separate Bibl file and URI but this raises the question of how we model bibliographic data (which we have just begun to think about).

For now @tacarlson please let us know if there is a problem in TEI file place/78. But Tom maybe wait on tweaking more until we are clear what needs to be done. Thanks!

**Comment by tacarlson** [comment]
I see two issues: 1) the lack of a space between "138-139" and "Map I B1, II B1, III." and 2) the lack of article citation.  Issue (1) is a visualization issue; issue (2) is a TEI issue.

I haven't yet put the abstract from GEDSH in 78.xml (the current abstract is my own test to see how to do certain features), and so I haven't encoded the article information in the TEI.  The "138-139" there was put in long before we decided to have a separate GEDSH article citation so that I could encode some confession information in the TEI.  Since 78.xml is curated by hand, I will modify it by hand to include the article information.

**Comment by davidamichelson** [comment]
Thanks, are you hand coding the article information on all of these? I see
that place/215 is missing its article author. Before you do too much by
hand maybe we should decide if we want to cite the articles as separate
bibl URIs (which would be my inclination).

**Comment by tacarlson** [comment]
No, I am only coding 3 places by hand, of which only Edessa has a GEDSH article.  For the others, I am entering the author name and article pages into the spreadsheet and generating the citation from that.  I'm 1/5 of the way through these, more or less.  I have not got to place/215, so the only article information it has is the headword, which was put in earlier.  The way to tell whether I've got to a place with GEDSH article author and pages is whether it has an abstract which is sourced to the GEDSH article.

Rather than entering the author name and article pages in the places spreadsheet, I could use XSL on the spreadsheet of GEDSH articles to generate a TEI bibl record for each GEDSH article, assigning URIs upward from our last used URI.  Then I could merely enter the URI for the article in the places spreadsheet (along with proofreading the GEDSH-derived abstract before I decide to use it, which takes time).

**Comment by davidamichelson** [comment]
Thanks for the elaboration, I think I would prefer the latter suggestion of
giving each article its own bibl.xml and URI (this will pay off for reasons
related to generating automated bibliographies on topics).

But since Tom has written the TEI to HTML transform a certain way for now,
perhaps we should discuss when to do this.

But I don't want you to spend too much time encoding something we will
discard ;)

Perhaps we can talk on Monday?

**Comment by tacarlson** [comment]
Any time Monday afternoon is fine for me.

**Comment by paregorios** [comment]
Guys, I'm afraid I'm slammed today.

**Comment by davidamichelson** [comment]
No worries, I'll talk to Thomas and we will report back.

**Comment by paregorios** [comment]
If there remains any work to do RE changing the encoding of bibliographic items, I think a new ticket (probably on the bibl repo) should be opened for it. I'm about to close this ticket, having dealt with the narrow XSLT issue of separating multiple 'citedRange' elements.
